src/components/Dashboard/Form/Team/TeamMembers/TeamMembers.js

```diff
@@ -19,16 +19,16 @@ const TeamMembers = () => (
         {fields.map((element, index) => (
           <div key={index} className={styles.TeamMembersInputWrapper}>
             <FieldWrapper
-              name={`${element}.name`}
+              name={`meeting.name`}
               component="input"
               componentSize="medium"
-              label={`Członek ${index + 1}`}
+              // label={`Członek ${index + 1}`}
```

Comment:
I guess if labels work properly based on `appContent` we should remove unnecessary lines.

Reply:
Sorry! My mistake!